src/Development/IDE/Spans/Type.hs

```diff
@@ -47,13 +50,15 @@ instance NFData SpanInfo where
 -- we don't always get a name out so sometimes manually annotating source is more appropriate
 data SpanSource = Named Name
                 | SpanS SrcSpan
+                | Lit SrcSpan String
```

Comment:
I couldn’t figure out where we actually make use of the `SrcSpan` that we store here. Can we just remove it?

We should probably also replace `String` by `Text`.

Reply:
I'll remove it. I was storing it as we store any other `SrcSpan`, but it turns out literals have to be excluded from all "go to definition" and "go to point" searches, so we end up not using it.